app/models/location.py

```diff
@@ -0,0 +1,26 @@
+from .. import db
+
+
+class Zipcode(db.Model):
+    __tablename__ = 'zip_codes'
+    id = db.Column(db.Integer, primary_key=True)
+    zip_code = db.Column(db.String(10), unique=True)
```

Comment:
Sorry @sandlerben, but I think it'd be better to change this back to a 5 character input. A lot of the models in the project are location based and we want this field to be unique, but I don't think we can mandate that each zip code inputted is a zip+4. For consistency, I think it's better to keep as just a 5 character zip. Let me know if you disagree.